test/canAddLikeTermPolynomialNodes.test.js

```diff
@@ -0,0 +1,16 @@
+const canAddLikeTermPolynomialNodes = require('../lib/checks/canAddLikeTermPolynomialNodes');
+
+const TestUtil = require('./TestUtil');
+
+function canBeAdded(expr, addable) {
+  TestUtil.testBooleanFunction(canAddLikeTermPolynomialNodes, expr, addable);
+}
+
+describe('Add like term polynomials', () => {
```

Comment:
'Can add like term polynomials' (you're not actually adding them, just seeing if you can)